app/stores/CollectionsStore.ts

```diff
@@ -21,6 +22,26 @@ export default class CollectionsStore extends Store<Collection> {
     super(rootStore, Collection);
   }
 
+  /**
+   * Return whether comments should be enabled, using the
+   * per-collection override if present, otherwise falling back
+   * to the team's workspace default.
+   */
+  canComment(collectionId?: string): boolean {
```

Comment:
Mostly the PR is straightforward, and I'm happy to include this functionality.

This method doesn't seem in quite the right place though– I think it would be better on the collection model `collection.canCreateComment` would work. Where is no `collectionId` available we should use the team preference as today.